### ISSUE TITLE
request: last sync time

### ISSUE THREAD

**Issue by zecoder** [ankidroid/Anki-Android#4051]
'Twould be nice to see the last sync date and time at the bottom of the main screen

**Comment by arooooon** [comment]
Yea I agree, sometimes you want to know if you really synced the tablet before you left for work.

Though it should be in settings next to sync, rather than on the main screen.

**Comment by timrae** [comment]
Since this feature isn't in anki desktop I think it's a bit outside the scope of AnkiDroid. You can address the root cause of the issue by using the sync intent. For example I have tasker do a sync every night when I plug my phone into charge.

I started working on a feature once that would check in the background whether or not you had unsynced changes and show a snack bar, but it would have required a change in the sync protocol to work properly unfortunately. Hopefully in the next major version of anki we'll get some improvements to sync.

**Comment by zecoder** [comment]
You don't need to have the time and place of the most recent sync, or a scan to see if there are unsynced changes from another sources, as that would be complex to implement. Just the last time that AnkiDroid synced as a very basic time stamp

**Comment by timrae** [comment]
Yeah I understand the feature request, I was just mentioning how I see the future development direction to address the issue.
